doc/holochain_101/src/development_conductor.md

```diff
@@ -0,0 +1,78 @@
+# Development Conductor
+
+Meant primarily for accelerating the development process, the easiest Conductor to run, with zero configuration required, is the `hc run` Conductor, which is built right into the [development command line tools](./intro_to_command_line_tools.md). It is useful for testing APIs or prototyping user interfaces, and it expects that you run the command from the directory valid DNA source files are contained. The command is simply:
+```shell
+hc run
+```
+
+This will start the DNA instance in a Conductor and open, by default, a WebSocket JSON-RPC server on port `8888`. How to use the API for this is covered in more depth in the [JSON-RPC interfaces article](./json_rpc_interfaces.md).
```

Comment:
```suggestion
This will start the DNA instance in a Conductor and open, by default, a WebSocket JSON-RPC server on port `8888`. You can find more details on how to use the API in your UI in the [JSON-RPC interfaces article](./json_rpc_interfaces.md).
```

@Connoropolous I notice I've been doing a bunch of converting your sentences from passive to active voice.  This change provides a good example. "How to X is Yed in the... " is in the passive voice.  "To X, verb Y" is in the active.  The more you remove the verb "to be" from your constructions the more lively they read.